Classes/LinphoneUI/UIMainBar.m

```diff
@@ -670,15 +670,16 @@ - (IBAction)videomailButtonAction:(UIButton *)sender {
 }
 
 - (IBAction)resourcesButtonAction:(UIButton *)sender {
-    ResourcesViewController *resourcesController = [[ResourcesViewController alloc] init];
-    float sysVer = [[[UIDevice currentDevice] systemVersion] floatValue];
-    
-    if (sysVer >= 8.0) {
-        [self showViewController:resourcesController sender:self];
-    }
-    else{
-        [self presentViewController:resourcesController animated:YES completion:nil];
-    }
+//    ResourcesViewController *resourcesController = [[ResourcesViewController alloc] init];
+//    float sysVer = [[[UIDevice currentDevice] systemVersion] floatValue];
+//    
+//    if (sysVer >= 8.0) {
```

Comment:
we should open Helper menu at first then from "Deaf / Hard of Hearing Resources" open Resources View